testing/py-scikit-learn/APKBUILD

```diff
@@ -0,0 +1,58 @@
+# Contributor: Carlos Giraldo <cgiraldo@gradiant.org>
+# Maintainer: Carlos Giraldo <cgiraldo@gradiant.org>
+pkgname=py-scikit-learn
+_pkgname=scikit-learn
+pkgver=0.20.3
+pkgrel=0
+pkgdesc="scikit-learn is a Python module for machine learning built on top of SciPy"
+url="http://scikit-learn.org/"
+arch="all"
+license="BSD"
+depends="py-numpy py-scipy"
+makedepends="py-setuptools py-numpy-dev python2-dev python3-dev"
+subpackages="py3-${pkgname#py-}:_py3 py2-${pkgname#py-}:_py2"
+source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
+builddir="$srcdir/$_pkgname-$pkgver"
+
+
+build() {
+	cd "$builddir"
+	python2 setup.py build
+	python3 setup.py build
+}
+
+package() {
+	mkdir -p "$pkgdir"
+	cd "$builddir"
+
+	local python; for python in python2 python3; do
+		$python setup.py install --prefix=/usr --root="$pkgdir"
+	done
+
+	install -D -m 644 COPYING \
+		"$pkgdir"/usr/share/licenses/$pkgname/LICENSE
+}
+
+_py() {
+	local python="$1"
+	pkgdesc="$pkgdesc ${python#python}"
+	depends="$depends $python"
+	install_if="$pkgname=$pkgver-r$pkgrel $python"
+
+	cd "$builddir"
+	$python setup.py install --prefix=/usr --root="$subpkgdir"
+}
+
+_py2() {
```

Comment:
Is there a reason to add a python2 package? Python2 will be EOL soon, so we prefer new packages to only provide python3 variants unless python2 is required for something.

Reply:
I agree. 
I just dropped the python2 package.